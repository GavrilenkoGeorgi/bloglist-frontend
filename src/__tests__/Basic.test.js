import React from 'react'
// import TestRenderer from 'react-test-renderer'
import ShallowRenderer from 'react-test-renderer/shallow'
import Basic from '../components/Basic'

// Basic Test with React-test-renderer
it('renders correctly react-test-renderer', () => {
	const renderer = new ShallowRenderer()
	renderer.render(<Basic />)
	const result = renderer.getRenderOutput()

expect(result).toMatchSnapshot()
})
