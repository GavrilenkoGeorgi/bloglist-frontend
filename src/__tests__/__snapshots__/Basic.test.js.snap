// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly react-test-renderer 1`] = `
<div>
  <h1>
     Basic Test
  </h1>
</div>
`;
